core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -185,20 +296,54 @@ void *impl_allocate_common(const int device_id,
 #ifndef CUDART_VERSION
 #error CUDART_VERSION undefined!
 #elif (defined(KOKKOS_ENABLE_IMPL_CUDA_MALLOC_ASYNC) && CUDART_VERSION >= 11020)
-  if (arg_alloc_size >= memory_threshold_g) {
-    error_code = cudaMallocAsync(&ptr, arg_alloc_size, stream);
-
-    if (error_code == cudaSuccess) {
-      if (stream_sync_only) {
-        KOKKOS_IMPL_CUDA_SAFE_CALL(cudaStreamSynchronize(stream));
+  // Always use the Async. Don't use the 40K Lower limit- may well be arch
+  // dependent.
```

Comment:
```suggestion
  // Always use the Async. Don't use the 40K Lower limit - may well be arch
  // dependent.
```
Is `memory_threshold_g` used anymore?